docs/setup/kubeedge_configure.md

```diff
@@ -95,14 +95,14 @@ modules:
 
 Refer [here](deploy-edge-node.md) to add edge nodes.
 
-#### Check if the certificates are created (Run on cloud side)
+#### Check the existence of certificates (cloud side)
 
 RootCA certificate and a cert/key pair is required to have a setup for KubeEdge. Same cert/key pair can be used in both cloud and edge.
 
-The cert/key will be generated in the /etc/kubeedge/ca and /etc/kubeedge/certs respectively, so this command should be run with root or users who have read/ write permission to those directories.
-We need to copy these files to the corresponding edge side server directory.
+cert/key should exist in /etc/kubeedge/ca and /etc/kubeedge/certs [Generate certs](https://github.com/kubeedge/kubeedge/blob/master/docs/setup/kubeedge_install_source.md#generate-certificates).
```

Comment:
```suggestion
cert/key should exist in /etc/kubeedge/ca and /etc/kubeedge/certs. Otherwise you may refer to [generate certs](https://github.com/kubeedge/kubeedge/blob/master/docs/setup/kubeedge_install_source.md#generate-certificates) to generate them.
```